src/loadsave.cpp

```diff
@@ -322,7 +325,29 @@ void Saver::PutText(const std::string &str, int length)
 
 /* When making any changes to saveloading code, don't forget to update the file 'doc/savegame.rst'! */
 
-static const uint32 CURRENT_VERSION_FCTS = 10;  ///< Currently supported version of the FCTS pattern.
+static const uint32 CURRENT_VERSION_FCTS = 11;  ///< Currently supported version of the FCTS pattern.
+
+PreloadData Preload(Loader &ldr)
+{
+	uint32 version = ldr.OpenPattern("FCTS");
+	if (version != 0 && (version < 10 || version > CURRENT_VERSION_FCTS)) ldr.version_mismatch(version, CURRENT_VERSION_FCTS);
+
+	PreloadData result;
+
+	if (version >= 11) {
+		result.timestamp = ldr.GetLongLong();
+		result.revision = ldr.GetText();
+		result.scenario_name = ldr.GetText();
+	} else {
+		result.timestamp = 0;
+		result.revision = "?";
+		result.scenario_name = "?";
```

Comment:
This `0` is likely where `1970` is coming from in the screenshot. While technically correct, wouldn't it be nicer if you showed "not available" or (`n/a` if there is no room)  in the display instead?

Here you may want to use `0` and an empty string for that (assuming scenario names are never empty).